### ISSUE TITLE
[@react-navigation/material-top-tabs][bare-react-app] Pressing on active tab causes the other tabs touch events to freeze

### ISSUE THREAD

**Issue by aprilmintacpineda** [react-navigation/react-navigation#10425]
### Current behavior

When I press on currently active tab, the other tabs' touch events will no longer work.the 

### Expected behavior

When I press on the currently active tab, the other tabs' touch events SHOULD STILL work.

### Reproduction

https://github.com/aprilmintacpineda/react-navigation-repro-tab-freezes

### Platform

- [X] Android
- [X] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [ ] @react-navigation/drawer
- [ ] @react-navigation/material-bottom-tabs
- [X] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack

### Environment

- [x] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               | ^6.0.8
| @react-navigation/material-top-tabs    | ^6.1.1
| react-native-safe-area-context         | ^4.1.2
| react-native-screens                   | ^3.13.1
| react-native-tab-view                  | ^3.1.1
| react-native-pager-view                | ^5.4.11
| react-native                           | 0.67.3
| node                                   | 16.13.0
| yarn                                   | 1.22.11

**Comment by ngxhuyhoang** [comment]
Same

**Comment by agusvazquez** [comment]
I am experiencing the same issue but with react navigation drawer

**Comment by reidgarner** [comment]
Same.

**Comment by joelbrewer** [comment]
Same. Any update on this?

**Comment by raajnadar** [comment]
Can confirm the code is perfect with this version.

"@react-navigation/native": "^6.0.6",
"@react-navigation/material-top-tabs": "^6.0.6",
"react-native-screens": "~3.10.1",
"react-native-tab-view": "^3.1.1",
"react-native-pager-view": "5.4.9",

Can someone create a runnable snack repro?? Would be easier to debug the issue.

**Comment by ElicaInc** [comment]
> Use "react-native-tab-view": "^2.16.0" will solve the problem

Thanks. This works for me.

**Comment by nadavdrewe** [comment]
Thanks this helped

**Comment by robertherber** [comment]
> Use "react-native-tab-view": "^2.16.0" will solve the problem

Yeah, it solves this issue 👍 However it's a bit glitchy in other ways (when resizing windows etc) - so would be great if compatibility with the later version was fixed 😃

**Comment by okwasniewski** [comment]
Hey @aprilmintacpineda,

Can you check if this issue still persists with the latest tab-view and pager-view versions?

**Comment by johnnywang** [comment]
Not the issuer opener, but can confirm that this is still happening with tab-view 3.3.2 and pager-view 6.1.2 (so not latest, but close)

**Comment by vargajacint** [comment]
The issue is still exist using:
- react-native-tab-view: 3.5.1,
- react-native-pager-view: 6.2.0,
- @react-navigation/material-top-tabs: 6.6.2,
- @react-navigation/native-stack: 6.9.12
- react-navigation/native: 6.1.6,

**Comment by phatmovista** [comment]
> The issue is still exist using:
> 
> * react-native-tab-view: 3.5.1,
> * react-native-pager-view: 6.2.0,
> * @react-navigation/material-top-tabs: 6.6.2,
> * @react-navigation/native-stack: 6.9.12
> * react-navigation/native: 6.1.6,

I can confirm the same as well

**Comment by johnnywang** [comment]
Bump on this, since it doesn't seem like this has been assigned out to anyone yet. We're in the process of upgrading Expo (and hence React and React Native), which seems like it might require a bump to the pager-view package? And based on the comments above, it looks like this is still an issue

**Comment by okwasniewski** [comment]
Hey @johnnywang, 

I will look into this early next week and try to reproduce this issue. Are you also seeing this on React Native app without expo?

**Comment by johnnywang** [comment]
@okwasniewski This required a pretty specific setup in our app to trigger, so I don't have an easy way to test or repro this outside of our Expo (bare) app. Maybe @vargajacint or @phatmovista can confirm?

**Comment by vargajacint** [comment]
Yes, I can confirm. This issue is still exist without Expo

**Comment by oguzydz** [comment]
This issue still exist, with React Native App without expo not bare app.

**Comment by Jun-FH** [comment]
I concur, the issue still exist.
Using the following, libraries and version:

@react-navigation/material-top-tabs: 6.6.3
react-native-pager-view: 6.2.1
 react-native-tab-view : 3.5.2
 
On our end a need to wait sometime(3-7 mins at least) by switching the tabs in real device(tested iPhone).
At some point the screen transitioning will freeze, yet in my end I can still tap on the tabs.

**Comment by johnnywang** [comment]
@okwasniewski I saw that there was a PR referencing this, that looks like it contained a fix: is this issue still open becuase it was an iOS-only fix?

**Comment by Jun-FH** [comment]
@johnnywang 
Looks like I made a mistake on the PR reference. Perhaps can remove the PR reference in the comment.
We refrain top tabs for now. We go with bottom tabs for now. 
We loss the transition animation but it looks good for our need at the moment. We may switch back to it later if needed.
Thank you for the follow up.

**Comment by ayelenguini** [comment]
any updates on this? the issue it still happening on latest version.

**Comment by great2lrn** [comment]
Any updates? 

This is also happening when swiping too fast between two screens. I may end up on Tab 2, but it displays the content of Tab 1.

**Comment by levepic** [comment]
I have a similar issue, found it yet only on IOS. Cannot reproduce it always. Having First and Second, Third tab, sometimes pressing on the screen inside Second tab captures the touch events on the First tab. So the view looks correctly as if you are on Second tab but whenever you touch (the menu inside it or empty space) it captures the touch events as if you pressed on the First tab screen. Both screens have menus (navigations) inside.  So if you touch the top of the screen the navigation event will happen which would happened if you touched the top of First tab etc. Making some navigations clear things up and it works fine again. Im using
`
      freezeOnBlur: true,
      popToTopOnBlur: true,` will try if it works without these but it doesnt happen all the time so hard to track.

Using the latest of everything at the momet


    "@react-navigation/bottom-tabs": "^7.2.0",
    "@react-navigation/elements": "^2.2.5",
    "@react-navigation/native": "^7.0.14",
    "@react-navigation/native-stack": "^7.2.0",
    "@react-navigation/stack": "^7.1.1",

    "react-native-screens": "^4.4.0",
